cli/omnibus/config/software/kontena-cli.rb

```diff
@@ -1,15 +1,16 @@
 name "kontena-cli"
-license "Apache 2.0"
+license "Apache-2.0"
+license_file "LICENSE.txt"
 default_version File.read('../VERSION').strip
-source path: "./wrappers"
+source path: ".."
```

Comment:
This seems to be a terrible idea, it ends up recursively copying the `../omnibus` to itself.